src/psyclone/parse/module_manager.py

```diff
@@ -243,6 +264,9 @@ def get_all_dependencies_recursively(self, all_mods):
         while todo:
             # Pick one (random) module to handle:
             module = todo.pop()
+            # Ignore any modules that we were asked to ignore
+            if module in self.ignores():
```

Comment:
Very observant. I had noticed that, and fixed it in the 'next' PR (to avoid modifying this while a review is in progress :) ), but I've backported it now :)